src/components/Gallery/index.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+
+import SwiperCore, { FreeMode, Navigation, Thumbs } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/react/swiper-react.js";
+
+import "swiper/swiper.scss";
+import "swiper/modules/navigation/navigation.scss";
+import "swiper/modules/pagination/pagination.scss";
+import "swiper/swiper.scss";
+import "./styles.scss";
+
+SwiperCore.use([FreeMode, Navigation, Thumbs]);
+
+function Gallery({ images }) {
+  const [thumbsSwiper, setThumbsSwiper] = useState(null);
+
+  return (
+    <div>
+      <Swiper
+        style={{
+          "--swiper-navigation-color": "#fff",
+          "--swiper-pagination-color": "#fff",
+        }}
+        loop={images.length > 3}
+        spaceBetween={10}
+        navigation={true}
+        thumbs={{ swiper: thumbsSwiper }}
+        className="mySwiper2"
+      >
+        {images.map(({ image: { alt, url } }) => (
+          <SwiperSlide key={url}>
+            <img src={url} alt={alt} />
+          </SwiperSlide>
+        ))}
```

Comment:
These lines repeat again from 46-50. Maybe store what the `.map` returns in a constant, so you can render that constant in these two places.